onnxruntime/core/mlas/lib/qgemm.cpp

```diff
@@ -354,10 +405,38 @@ Return Value:
     const size_t BufferAlignment = MlasGetPreferredBufferAlignment();
     const size_t AlignedBytesRequired = (BytesRequired + BufferAlignment - 1) &
         ~(BufferAlignment - 1);
+    //If this gemm B argument is used in a dynamically quantization gemm operation we can optimize for
+    //this use case. Concat both packed representations for later decision.
```

Comment:
could you elaborate more on this use case? why would one call `MlasGemmPackBSize` for running `MlasDynamicQgemm` functions?